acpi/src/address.rs

```diff
@@ -6,7 +6,7 @@ use core::convert::TryFrom;
 
 /// This is the raw form of a Generic Address Structure, and follows the layout found in the ACPI tables. It does
 /// not form part of the public API, and should be turned into a `GenericAddress` for most use-cases.
-#[derive(Clone, Copy, Debug)]
+#[derive(Debug, Clone, Copy)]
```

Comment:
I'm a big fan of styling and ordering consistency, but if you're of the mind this isn't worth pursuing then I'll undo the changes. I figured they would be a relatively minor & welcome cleanup.

Reply:
Cleanups are always welcome, but generally they're nicer in a separate commit or even PR just to make it easier to review the changes to API etc. (I find having a nice git history invaluable for `acpi` because I work on it relatively infrequently, and it's difficult to remember your thoughts when changes were made).

I also unfortunately have too-strong views on the ordering of derived traits - I much prefer having `Clone, Copy, Debug` than the new ordering, but when you have multiple contributors it very quickly gets dwarfed by the importance of just getting stuff merged eventually. However, if we're going to change them to be consistent on one style, I'd much prefer the existing one.

Reply:
Sure thing, I've undone the styling changes. May make a PR for more pervasive code style cleanups before the next full release, whenever that is.